### PR TITLE
user TapBreadcrumbDataSource in the session message

### PR DESCRIPTION
**Pull request by nelsitoPuglisi**
## Goal

Sends taps as span events of the SessionSpan instead of breadcrumbs.

## Testing

Unit tests, Integration tests and Manually tested.

### CODE REVIEW

**Comment by nelsitoPuglisi** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/embrace-io/embrace-android-sdk/665?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#665** <a href="https://app.graphite.dev/github/pr/embrace-io/embrace-android-sdk/665?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#663** <a href="https://app.graphite.dev/github/pr/embrace-io/embrace-android-sdk/663?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#662** <a href="https://app.graphite.dev/github/pr/embrace-io/embrace-android-sdk/662?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @nelsitoPuglisi and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by bidetofevil — CHANGES_REQUESTED** [review]
I think we need to keep the v1 implementation working until this is released and we know that v2 is working completely

File: embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/payload/Breadcrumbs.kt [hunk 1]
```diff
@@ -21,12 +21,6 @@ internal data class Breadcrumbs(
     @Json(name = "vb")
     val viewBreadcrumbs: List<ViewBreadcrumb>? = null,
 
-    /**
-     * List of breadcrumbs which relate to screen taps.
-     */
-    @Json(name = "tb")
```
[the remark this answers is not in the document]
Comment (bidetofevil) [reply embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/payload/Breadcrumbs.kt:27]: As discussed off line, this should be fine

**Review by bidetofevil — APPROVED** [review]
LGTM! Sorry for the confusion

**Review by fractalwrench — APPROVED** [review]
LGTM

**Pull request merged**